docs/src/pages/components/data-grid/demo/FullFeauredDemo.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { XGrid, GridToolbar } from '@material-ui/x-grid';
+import {
+  useDemoData,
+  getCommodityColumns,
+  getEmployeeColumns,
+  getRealData,
+} from '@material-ui/x-grid-data-generator';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import { makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Button from '@material-ui/core/Button';
+import FormLabel from '@material-ui/core/FormLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: 400,
+    width: '100%',
+    '& .MuiFormGroup-options': {
+      alignItems: 'center',
+      justifyContent: 'center',
+      marginBottom: 15,
+      '& fieldset': {
+        padding: `0 10px`,
+      },
+      '& button': {
+        marginLeft: 20,
+      },
+    },
+  },
+});
+
+const SettingsPanel = ({ onApply, type, size }) => {
```

Comment:
While we might not have a convention for the source, for the docs, we are always using this structure:

```suggestion
function SettingsPanel(props) {
  const { onApply, type, size } = props;
```